scraper_common.js

```diff
@@ -75,6 +78,33 @@ async function execute(usePuppeteer, scrapers) {
                     return result;
                 });
             results.push(returnValue);
+            // Coerce the results into the format we want.
+            let returnValueArray = [];
+            if (Array.isArray(returnValue)) {
+                returnValueArray = returnValue;
+            } else if (returnValue) {
+                returnValueArray = [returnValue];
+            }
+            // Write the data to FaunaDB.
+            if (WRITE_TO_FAUNA) {
+                await Promise.all(
+                    returnValueArray.map(async (res) => {
+                        await dbUtils.writeScrapedData({
+                            name: res.name,
+                            street: res.street,
+                            city: res.city,
+                            zip: res.zip,
+                            availability: res.availability,
+                            hasAvailability: res.availability,
+                            extraData: res.extraData,
+                            timestamp: moment().utc().format(),
+                            signUpLink: res.signUpLink,
+                            massVax: res.massVax,
+                            siteTimestamp: res.siteTimestamp,
```

Comment:
@harcod are there any other fields I should include, that you can spot?

Reply:
synced with Rob offline. looked in `appointmentData.service.js` and saw that I should add the `restrictions` field which I've done.